src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js

```diff
@@ -206,12 +216,12 @@ function paymentsDetails(state) {
  * @function
  * @description Initializes Adyen Checkout My Account events
  */
-function initializeAccountEvents() {
+async function initializeAccountEvents() {
```

Comment:
Is async/await working properly with SiteGenesis?

Reply:
yes, it will depend on the browser. All browsers support it except for IE, which is getting deprecated. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#browser_compatibility